docs/structure.md

```diff
@@ -19,59 +15,21 @@ the tenant cluster for its own particular cluster scope. This implies to not
 create a Kubernetes client for the tenant cluster when executing the first
 action of a given cluster scope.
 
-```
+```nohighlight
 ├── cmd
-│   └── cl001
-│       └── ac001
-│           ├── execute
-│           └── explain
+│   └── cl001
+│       └── ac001
+│           ├── execute
+│           └── explain
 └── pkg
     └── action
-        └── ac001
+        └── ac001
 ```
 
-```
-$ awscnfm cl001 ac001 execute
-$ awscnfm cl001 ac001 explain
-$ awscnfm cl001 ac002 ...
-```
-
-
+The according subcommand hierarchy:
```

Comment:
Should does not be moved to line 17?

Reply:
I'm adding a sub headline `The package hierarchy` to line 17. Then it makes sense like this, IMO.